conf/vpc/policies/example-policy.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Version": "2008-10-17",
+  "Statement": [
+	{
+	  "Sid": "",
+	  "Effect": "Allow",
+	  "Principal": "*",
+	  "Action": "*",
```

Comment:
Nope. These are copied exactly from what AWS is using for our policies. My bet is because the Statement is an Array you would have to specify different combinations separately. Because the API is stringly typed on this one, I'm just using the exact format that we are getting from them :(